src/test/jtx/impl/Env.cpp

```diff
@@ -566,8 +558,21 @@ Env::do_rpc(
     std::vector<std::string> const& args,
     std::unordered_map<std::string, std::string> const& headers)
 {
-    return rpcClient(args, app().config(), app().logs(), apiVersion, headers)
-        .second;
+    auto response =
+        rpcClient(args, app().config(), app().logs(), apiVersion, headers);
+
+    for (unsigned ctr = 0; (ctr < retries_) and (response.first == rpcINTERNAL);
```

Comment:
This should be `(ctr < (retries_-1))`, since there is an initial attempt outside of the loop on line 561.

Reply:
The previous is "attempt", and these are "retries" :-).  Actually there are some tests that doesn't require retries cause they expect error. `env.set_retries(0)` they used is understandable way to show that there will not be any retries after first connect attempt.  Setting `env.set_retries(1)` feels uncomfortable. Add more logic for this is unreasonable.